pkg/reconciler/common/targetnamespace.go

```diff
@@ -0,0 +1,54 @@
+package common
+
+import (
+	"context"
+	mf "github.com/manifestival/manifestival"
+	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"os"
+	"path/filepath"
+)
+
+func CreateTargetNamespace(labels map[string]string, obj v1alpha1.TektonComponent, kubeClientSet kubernetes.Interface) error {
+	namespace := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: obj.GetSpec().GetTargetNamespace(),
+			Labels: map[string]string{
+				"operator.tekton.dev/targetNamespace": "true",
```

Comment:
do we need to set owner reference as well on the newly created namespace?

Reply:
cc @nikhil-thomas @PuneetPunamiya @savitaashture

Reply:
That will be ideal.
@PuneetPunamiya shall we add the creting CR (TektonPipeline) or (TektonConfig) as the ownerReference

Reply:
yep I'll add it

Reply:
👍🏻

Reply:
updated 🙂